src/app/containers/Article/index.jsx

```diff
@@ -48,15 +49,15 @@ class ArticleContainer extends Component {
       const data = await response.json();
       const amp = isAmpPath(path);
 
-      return { amp, data };
+      return { amp, data, service };
     } catch (error) {
       console.log(error); // eslint-disable-line no-console
       return {};
     }
   }
 
   render() {
-    const { amp, data } = this.props;
+    const { amp, data, service } = this.props;
```

Comment:
`service` needs to be included in `ArticleContainers`' propTypes.

Reply:
@jtart I've addressed the issues that you've raised.